storybook/constants/siteConstants.js

```diff
@@ -19,7 +19,7 @@ export const DOCUMENTATION_URL = {
   PATTERNFLY_ORG_CARDS: `${BASE_URL.PATTERNFLY_ORG}pattern-library/cards/`,
   PATTERNFLY_ORG_INLINE_EDIT: `${
     BASE_URL.PATTERNFLY_ORG
-  }forms-and-controls/inline-edit/`,
+  }pattern-library/forms-and-controls/inline-edit/`,
```

Comment:
Better to change from `BASE_URL.PATTERNFLY_ORG` to `BASE_URL.PATTERNFLY_LIBRARY` then to add `pattern-library/`.

And while there could you update `PATTERNFLY_ORG_CARDS` as well.

Reply:
Please review updates changes.